dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -217,31 +269,29 @@ private void download(String urlLocation, String dLocation, String groupId, Stri
         }
     }
 
-    private boolean checksumIsAvailable(String urlLocation, String checksumFormat) {
+    private boolean checksumIsAvailable(String urlLocation, String checksumFormat, HashMap<String, String> checkSumCache) {
         boolean result = true;
         try {
-            ArtifactDownloaderUtils.getMasterChecksum(urlLocation, checksumFormat);
+            if (checkSumCache.get(checksumFormat) != null)
```

Comment:
assuming keys cannot be mapped to null, use `checkSumCache.containsKey()` instead